### ISSUE TITLE
Assign GitLab MRs on creation and not as update

### ISSUE THREAD

**Issue by fzoske** [renovatebot/renovate#9252]
**What would you like Renovate to be able to do?**
Assign the MRs in GitLab directly to one of the assinees on creation instead of updating the MR.

In the emails generetad by GitLab currently only contain: "Assignee changed to <USER>" but it would be great to have the description of the MR in the mail as well.

Best regards,
Fabian

**Comment by viceice** [comment]
This would need a general refactoring, as renovate currently don't assign any reviewers (beside some platform defaults) to any pr.

**Comment by fzoske** [comment]
@viceice But renovate currently assigns the prs to the configured assignees, but at a later point.

**Comment by viceice** [comment]
@fzoske Yes, it's a completly separate function for all renovate supported platforms

https://github.com/renovatebot/renovate/blob/74d7691befa635c1220d8569936273bfb950e748/lib/platform/types.ts#L152

**Comment by rarkins** [comment]
I don't have any objection to assigning at the time of creation and saving an API call while we're at it, but we need to work out how to do that elegantly so that platforms which don't support it in an extra operation can do so. e.g. maybe pass assignees and reviewers to the createPr function in platform and let the platform decide whether it can add them directly with the PR creation or do a second query after.
